### PR TITLE
Upgrade flutter to 3.19

### PR DESCRIPTION
**Pull request by micklenormand**

### CODE REVIEW

File: generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -416,13 +542,14 @@
...
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				DEVELOPMENT_TEAM = UPPST333D7;
 				ENABLE_BITCODE = NO;
 				INFOPLIST_FILE = Runner/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER =  <%= applicationPrefix %>.<%= applicationName %>;
+				PRODUCT_BUNDLE_IDENTIFIER = com.example.myapp;
```
Comment (jvasseur) [inline generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj:552]: Why removing the tempalted vars here ?

File: generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -438,13 +565,14 @@
...
```
Comment (jvasseur) [inline generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj:575]: Same here.

**Review by jvasseur — CHANGES_REQUESTED** [review]

File: generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (micklenormand) [reply generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj]: fixed :-)

Comment (micklenormand) [reply generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj]: fixed :-)

Comment (micklenormand) [reply generators/flutter-mobile/templates/base/ios/Runner.xcodeproj/project.pbxproj]: fixed :-)

**Pull request merged**